src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -0,0 +1,26 @@
+package juja.sqlcmd;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DatabaseManager {
+    private static final String PORT = "5432";
+    private static final  String SERVER = "localhost";
+    private Connection connection;
```

Comment:
Pay your attention on whitespaces (http://bit.ly/2j2qPE1)

Reply:
Thanks for your remark.